internal/services/datafactory/data_factory_linked_service_azure_blob_storage_resource_test.go

```diff
@@ -423,3 +438,28 @@ resource "azurerm_data_factory_linked_service_azure_blob_storage" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, tenantID)
 }
+
+func (LinkedServiceAzureBlobStorageResource) insecureConnectionString(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-df-%d"
+  location = "%s"
+}
+
+resource "azurerm_data_factory" "test" {
+  name                = "acctestdf%d"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+}
+
+resource "azurerm_data_factory_linked_service_azure_blob_storage" "test" {
+  name                        = "acctestlsblob%d"
+  data_factory_id            = azurerm_data_factory.test.id
+  insecure_connection_string = "DefaultEndpointsProtocol=https;AccountName=foo;AccountKey=bar"
```

Comment:
can we add a test that uses both?z	
```suggestion
  insecure_connection_string = "DefaultEndpointsProtocol=https;AccountName=foo;"
    connection_string = "AccountKey=bar"
```

Reply:
Unfortunately, we can't as we can only do one or the other. ConnectionString is an `interface{}` in the sdk which allows for you to send the connection string as a SecureString or a regular String